golem-cli/tests/api_deployment_http_handler.rs

```diff
@@ -95,16 +95,16 @@ async fn api_deployment_http_handler(
         client
             .post(format!("http://{host}/test?foo=baz"))
             .header("test-header", "test-header-value")
-            .body("\"test-body\"")
+            .body("test-body")
             .send()
             .await
     }?;
     assert_eq!(res.status().as_u16(), 200);
     assert_eq!(
         res.headers().get("echo-test-header").unwrap(),
-        "\"test-header-value\""
+        "test-header-value"
```

Comment:
This is covering the non-json body in the pr title.
Assuming the existing behavior is reasonably covered, I don't think further tests are necessary